pandas/tests/extension/test_datetime.py

```diff
@@ -0,0 +1,237 @@
+import numpy as np
+import pytest
+
+from pandas.core.dtypes.dtypes import DatetimeTZDtype
+
+import pandas as pd
+from pandas.core.arrays import DatetimeArrayMixin as DatetimeArray
+from pandas.tests.extension import base
+
+
+@pytest.fixture(params=["US/Central"])
+def dtype(request):
+    return DatetimeTZDtype(unit="ns", tz=request.param)
+
+
+@pytest.fixture
+def data(dtype):
+    data = DatetimeArray(pd.date_range("2000", periods=100, tz=dtype.tz),
+                         dtype=dtype)
+    return data
```

Comment:
Do the tests also pass with a tz-naive DatetimeArray?

Reply:
No, because `is_extension_dtype(DatetimeArray[datetime64[ns]].dtype)` is false.

For tz-naive DatetimeArray.dtype is just a NumPy type.

For testing, we could consider adding a dummy DatetimeDtype for tz-naive data... but that might require modifying the code a bit...